modules/nf-core/toulligqc/main.nf

```diff
@@ -0,0 +1,35 @@
+
+process TOULLIGQC {
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/toulligqc:2.5.2--pyhdfd78af_0':
+        'biocontainers/toulligqc:2.5.2--pyhdfd78af_0' }"
+
+    input:
+    path seq_summary
```

Comment:
looks like from the doc the tool can also accepts BAM and FastQ files. Can you add this?

Reply:
Thank you for the review! I have committed your suggestions and will work on implementing FASTQ and ABM files to the module.